config/v1/types_cluster_version.go

```diff
@@ -408,6 +424,15 @@ type URL string
 // Update represents an administrator update request.
 // +k8s:deepcopy-gen=true
 type Update struct {
+	// architecture is an optional field that indicates the desired
+	// value of the cluster architecture. In this context cluster
+	// architecture means either homogeneous or heterogeneous. When
+	// this field is part of spec, architecture is ignored if image is
```

Comment:
> When this field is part of spec...

We can drop this, because "Update represents an administrator update request." scoping on the type a few lines up, and:

```console
$ git --no-pager grep 'Update `json:' origin/pr/1339 config/v1/types_cluster_version.go
origin/pr/1339:config/v1/types_cluster_version.go:      DesiredUpdate *Update `json:"desiredUpdate,omitempty"`
origin/pr/1339:config/v1/types_cluster_version.go:      ConditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
```

mean that `Update` will only ever be part of the `spec` or other locations that represent admin requests.